blobspike/README.md

```diff
@@ -0,0 +1,198 @@
+# Azure Blob Storage Copy
```

Comment:
I suggest revisiting the structure of the document. I have issues following which sections are related to one another. Perhaps it would be good to provide a table summarizing options with duration, pro and cons?